### PR TITLE
[New Rule] Registry Hive File Creation via SMB

### PR DESCRIPTION
**Pull request by Samirbous**
Identifies the creation or modification of a medium size registry hive file via the SMB protocol :

image

### CODE REVIEW

**Review by w0rk3r — APPROVED** [review]
LGTM

**Review by DefSecSentinel — APPROVED** [review]
LGTM

File: etc/non-ecs-schema.json [hunk 1]
```diff
@@ -49,7 +49,8 @@
   },
   "logs-endpoint.events.*": {
     "process.Ext.token.integrity_level_name": "keyword",
-    "process.parent.Ext.real.pid": "long"
+    "process.parent.Ext.real.pid": "long",
+	"file.Ext.header_bytes": "keyword"
```
Comment (brokensound77) [inline etc/non-ecs-schema.json:53]:
```suggestion
    "file.Ext.header_bytes": "keyword"
```

File: rules/windows/credential_access_moving_registry_hive_via_smb.toml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+rule_id = "a4c7473a-5cb4-4bc1-9d06-e4a75adbc494"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Lateral Movement", "Credential Access"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+file where event.type == "creation" and
+ /* regf file header */
+ file.Ext.header_bytes : "72656766*" and file.size >= 30000 and
```
Comment (brokensound77) [inline rules/windows/credential_access_moving_registry_hive_via_smb.toml:29]: Can you add an eql comment on what those bytes are

**Review by brokensound77 — APPROVED** [review]

**Pull request merged**